### DISCUSSION TITLE
Add Discord Username support to profiles!

### DISCUSSION THREAD

**Discussion by Animan8000** [community/community#58447] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There is the option to add links/usernames from other platforms (Twitter, Twitch, YouTube, etc.) but not for Discord usernames. Why? I don't see a reason not to be able to add your Discord username to your profile.

**Comment by the-officialjosh** [comment]
It seems that Discord not displaying may be an issue with affiliations, as even on TikTok, the previous Instagram and YouTube links are no longer present on the profile. While I am not familiar with the inner workings of these companies, it is likely that a mutual agreement or partnership is required for the inclusion of such link types.
